jupyterhub/jupyterhub/base/jupyterhub-dc.yaml

```diff
@@ -98,6 +98,17 @@ spec:
         volumeMounts:
         - mountPath: /opt/app-root/configs
           name: config
+        readinessProbe:
```

Comment:
Will the DC ever succeed if 2 out of 3 pods never fulfill the readinessProbe? Aren't we gonna end up with failing deployments?

Thoughts @maulikjs ?

Reply:
Yeah it works perfectly, the DC just waits for the first pod to come online and then just starts the other 2 and completes.

![image](https://user-images.githubusercontent.com/22432723/126648841-fb0ae68f-2685-4b68-a32e-6adbd91a67bb.png)